### PR TITLE
Added check and Warning for Uninstalling BSIPA

### LINKED ISSUE #72: Crash during attempt of removing the mods/BSIPA
When IPA.exe is missing from the folder Mod Assistant will crash on the attempt of checking its version.
It happens during "Remove All Mods" or "Uninstall BSIPA" action.

To reproduce:
- Delete IPA.exe manually
- Run Mod Assistant
- In Options click one of the buttons - "Remove All Mods" or "Uninstall BSIPA"

I've noticed it after 1.6.1 released and "Install or Update" button was grayed out.
So the solution was to Change the game version to 1.6.0 and refreshing the page, so the button got active again letting you reinstall the IPA so it can be removed.

screenshot 1578526663

### PR DESCRIPTION
**Pull request by DomDom3333**
Simple check if IPA.exe and IPA Directory exists before trying to delete it. This should prevent crashes. Closes #72

### CODE REVIEW

File: ModAssistant/Pages/Mods.xaml.cs [hunk 1]
```diff
@@ -693,8 +693,15 @@ public void UninstallMod(Mod mod)
                     break;
                 }
             }
-            if (mod.name.ToLower() == "bsipa")
-                UninstallBSIPA(links);
+            if (mod.name.ToLower() == "bsipa") { 
+                if (Directory.Exists(Path.Combine(App.BeatSaberInstallDirectory, "IPA")) & File.Exists(Path.Combine(App.BeatSaberInstallDirectory, "IPA.exe"))){
```
Comment (lolPants) [inline ModAssistant/Pages/Mods.xaml.cs:697]:
```suggestion
                if (Directory.Exists(Path.Combine(App.BeatSaberInstallDirectory, "IPA")) && File.Exists(Path.Combine(App.BeatSaberInstallDirectory, "IPA.exe"))){
```

File: ModAssistant/Pages/Mods.xaml.cs [hunk 2]
```diff
@@ -693,8 +693,15 @@ public void UninstallMod(Mod mod)
...
+                    UninstallBSIPA(links);
+                }
+                else
+                {
+                    System.Windows.Forms.MessageBox.Show("BSIPA was not uninstalled as it was not found.", "Failed to uninstall BSIPA.", MessageBoxButtons.OK,MessageBoxIcon.Warning);
```
Comment (lolPants) [inline ModAssistant/Pages/Mods.xaml.cs:702]: This should be localized

File: ModAssistant/Pages/Mods.xaml.cs (hunk 1 above)
Comment (lolPants) [reply ModAssistant/Pages/Mods.xaml.cs:697]: This should be a conditional AND, not a logical AND

**Review by lolPants — CHANGES_REQUESTED** [review]

File: ModAssistant/Pages/Mods.xaml.cs (hunk 1 above)
Comment (DomDom3333) [reply ModAssistant/Pages/Mods.xaml.cs:697]: Shouldn't it check for both?

Comment (lolPants) [reply ModAssistant/Pages/Mods.xaml.cs:697]: Yes, while a logical AND (single &) *will* work, it's not best practice. A conditional AND (double &&) is recommended to minimise introducing unintended behaviour.

**Review by lolPants — APPROVED** [review]

**Pull request merged**